ui/src/components/credentials/IssuanceMethodForm.tsx

```diff
@@ -25,32 +24,23 @@ import { useEnvContext } from "src/contexts/Env";
 import { Connection } from "src/domain";
 import { AsyncTask, isAsyncTaskDataAvailable } from "src/utils/async";
 import { makeRequestAbortable } from "src/utils/browser";
-import {
-  ACCESSIBLE_UNTIL,
-  CREDENTIAL_LINK,
-  DID_SEARCH_PARAM,
-  VALUE_REQUIRED,
-} from "src/utils/constants";
+import { ACCESSIBLE_UNTIL, CREDENTIAL_LINK, VALUE_REQUIRED } from "src/utils/constants";
 import { processZodError } from "src/utils/error";
 
 export function IssuanceMethodForm({
+  did,
```

Comment:
'did' is defined but never used.

Is your linter working?

Reply:
Looks like the pre-commit hook is not working